### DISCUSSION TITLE
I didn't got lib/forge-std folder when i installed foundry Please help

### DISCUSSION THREAD

**Discussion by ayesparshh** [Cyfrin/foundry-full-course-cu#2071] [Q&A] [answered]
this is my vscode file directories
![Screenshot 2024-07-22 031453](https://github.com/user-attachments/assets/1853960c-1424-452d-aa11-827a853c8145)

this is from video
![Screenshot 2024-07-22 031440](https://github.com/user-attachments/assets/c0237455-a8f9-4bd4-8049-df089e5522ae)

**Comment by EngrPips** [comment]
If you initiate the repo with `forge init` you are supposed to get `lib/forge-std`

**Comment by Shiva-Sai-ssb** [comment, marked as answer]
It's weird that after initializing a Foundry project by doing `forge init`, the project did not install the `forge-std` library by default. I would suggest try to initialize a new project and see if it installs `forge-std` properly. 

Also, make sure you are using the latest version of Foundry. You can update to the latest version by running the command `foundryup` in the terminal.

If the issue still persists, you can install it manually. Create a new folder named lib, run the command `forge install foundry-rs/forge-std` in the terminal. This will install the forge-std library.

**Reply by EngrPips** [reply]
indeed

**Comment by ayesparshh** [comment]
Thank you it is now solved

**Reply by Shiva-Sai-ssb** [reply]
You're welcome. Keep going with the course! :)

**Comment by EngrPips** [comment]
Awesome, glad you figured it out. Keep crushing it.
